### PR TITLE
upgrade to pulpcore 3.11 + Proxy username and password are now separated fields.

### PR DESCRIPTION
**Pull request by rochacbruno**
- Reverted the PR that splitted those fields
- Deleted utils added for that PR
- Pulcore updated to 3.11
- Adjusted serializers
- Fixed test cases
- make requirements
- removed ALLOWED_CONTENT_CHECKSUM settings

Issue: AAH-425

### CODE REVIEW

File: requirements/requirements.insights.txt [hunk 1]
```diff
@@ -181,7 +181,7 @@ pulp-ansible==0.7.1
     # via galaxy-ng (setup.py)
 pulp-container==2.3.1
     # via galaxy-ng (setup.py)
-pulpcore==3.10.0
+pulpcore==3.11.0
```
Comment (awcrosby) [inline requirements/requirements.insights.txt:184]: probably need to run `make requirements` again as 3.11.0.post0 has been released

**Review by awcrosby — CHANGES_REQUESTED** [review]
We need to remove this as part of upgrade to 3.11:
https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/settings.py#L82

File: requirements/requirements.insights.txt (hunk 1 above)
Comment (rochacbruno) [reply requirements/requirements.insights.txt]: @awcrosby done!

**Review by awcrosby — APPROVED** [review]
LGTM

**Review by newswangerd — APPROVED** [review]

**Pull request merged**